### DISCUSSION TITLE
Bungeecord Chat Colour

### DISCUSSION THREAD

**Discussion by Harvle** [Zrips/CMI#2253] [Feature Requests]
### Description of feature/improvement request: ###
This would simply add a chat colour menu which will also be compatible with bungeecord.
### Detailed explanation of what it should do, when and in what situation: ###
A player may type /cmi chatcolor or just /chatcolor, and it would bring up a menu where a player can select a color for the chat, as well as the modifiers such as bold, italic etc. The permissions would respect the cmi.colors.chat.[color]. This would then have an option in config to be saved to the bungeecord server, where a player will then have their chat color synced across all servers (or through SQL once it has been improved to support multiple servers fully, as other servers tend to overwrite eachothers data with SQL currently). A similar system may also be implemented for carrying across nicknames, too.
### Basic explanation as to why this feature is needed and/or useful to have: ###
CMI has many chat based features, but doesn't contain anything to do with a player configurable chat colour. I feel that it would in a way "complete" the chat section of CMI, as it will allow people to remove 1 more plugin that manages chat and use CMI for everything chat related.

**Comment by PryPurity** [comment]
Yes. I would love to see chat color being implamented, not just for bungee, for but for the main plugin itself.
